tests/zfs-tests/tests/functional/cli_root/zfs_mount/Makefile.am

```diff
@@ -19,7 +19,8 @@ dist_pkgdata_SCRIPTS = \
 	zfs_mount_all_mountpoints.ksh \
 	zfs_mount_encrypted.ksh \
 	zfs_mount_remount.ksh \
-	zfs_multi_mount.ksh
+	zfs_multi_mount.ksh \
+	zfs_mount_test_race.sh
```

Comment:
Can you place this after `zfs_mount_remount.ksh` in order to keep the tests in alphabetical order.  It'd be nice to order the `linux.run` file similarly too, it appears to also be out of order.